alexandria/settings/alexandria.py

```diff
@@ -220,3 +220,11 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
 ALEXANDRIA_CONTENT_SEARCH_TYPE = env.str(
     "ALEXANDRIA_CONTENT_SEARCH_TYPE", default="phrase"
 )
+
+# Mime types that are considered safe for Content-Disposition: inline
+SAFE_FOR_INLINE_DISPONSITION = [
```

Comment:
this seems to be a typo

```suggestion
SAFE_FOR_INLINE_DISPOSITION = [
```

Reply:
Thanks, fixed!